### ISSUE TITLE
Search would be more useful if it searched inside partitions (PowerQuery code) as well

### ISSUE THREAD

**Issue by WayneGibbings** [TabularEditor/TabularEditor3#663]
For example - I want to find if a certain table is referenced inside a model that has dozens of queries - it would be useful if I could open Tabular Editor and use the Find function to search across the entire model including the PowerQuery code that is visible in the partitions

**Comment by otykier** [comment]
You're right, and in fact it was always the intention that selection "Expressions" in the Find options dropdown, should also allow searching partition queries. As a workaround, until we get this fixed, use Dynamic LINQ. I.e. to find a partition that contains the word "select", type the following in the search box:

```
:Query.Contains("SELECT", StringComparison.OrdinalIgnoreCase)
```

**Comment by otykier** [comment]
Fixed in [3.3.6](https://github.com/TabularEditor/TabularEditor3/releases/tag/3.3.6)!
